services/tasks-svc/migrations/000020_organization_ids.down.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE patients
+	ADD COLUMN IF NOT EXISTS
+	organization_id uuid NOT NULL;
```

Comment:
I doubt you can just add a column and constrain it NOT NULL without providing a value to set